lib/matplotlib/cbook.py

```diff
@@ -2192,6 +2192,21 @@ def is_math_text(s):
     return even_dollars
 
 
+def _check_1d(x):
+    '''
+    Converts a sequence of less than 1 dimension, to an array of 1
+    dimension; leaves everything else untouched.
+    '''
```

Comment:
This breaks anything using units that relies upon having `atleast_1d` called; this includes the "simple" unit framework in the examples.